ref.yml

```diff
@@ -392,29 +397,113 @@ fields:
       used:
         - coq.opam
         - .travis.yml
-  - tested_coq_nix_versions:
+  - cachix:
+      used:
+        - nix-action.yml.nix
       type: list
-      description: If empty, the master branch of Coq will be tested.
+      description: Cachix repositories to use
       item_fields:
-        - version_or_url:
+        - name:
             required: true
-            description: >
-              This can be a version number of a version packaged in
-              nixpkgs (example: 8.5, 8.11).  Cf. the coqPackages_8_XX
-              attributes defined at the end of this file:
-              https://github.com/NixOS/nixpkgs/blob/nixpkgs-unstable/pkgs/top-level/coq-packages.nix
-              (Quote the version number, otherwise '8.10' becomes '8.1'.)
-
-              Or this can be a URL, to test your project with a
-              version of Coq under development.  Typically this is
-              https://github.com/coq/coq-on-cachix/tarball/master but
-              'master' may be replaced by any valid reference like one
-              of the branches available at
-              https://github.com/coq/coq-on-cachix/branches or a valid
-              commit hash.
+            description: Name of the cachix repository
+            examples:
+              - "coq"
+              - "math-comp"
+        - key:
+            required: false
+            description: Secret signingKey for the repository
+        - token:
+            required: false
+            description: Secret authToken for the repository
```

Comment:
Providing an example would avoid people thinking that they need to put actual secrets in `meta.yml`.